tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py

```diff
@@ -365,13 +364,6 @@ def publish(
     return all(context.state is ContextState.SUCCESSFUL for context in publish_connector_contexts)
 
 
-def validate_publish_options(pre_release: bool, context_object: Dict[str, Any]):
-    """Validate that the publish options are set correctly."""
-    for k in ["spec_cache_bucket_name", "spec_cache_gcs_credentials", "metadata_service_bucket_name", "metadata_service_gcs_credentials"]:
-        if not pre_release and context_object.get(k) is None:
-            click.Abort(f'The --{k.replace("_", "-")} option is required when running a main release publish pipeline.')
```

Comment:
I think [this code](https://github.com/airbytehq/airbyte/pull/28067/files?diff=split&w=0#diff-956ec778a0cbdb8fbf60e4327fc8b80c3c8f06739870ad4f56c5e822203464bfR236-R263) needs to be updated to set it to required before removing this validation